Resources/Prototypes/_Moffstation/PersonalItems/Items/Cooks-the-Meth/Plushiescooksplushie.yml

```
@@ -0,0 +1,25 @@
+# Player: huaqas - Character: Cooks-The-Meth
+- type: entity
+  parent: PlushieLizard
+  id: PersonalItemCooksPlushie
+  name: Cooks plushie
+  description: A bright orange plushie resembling the chemist, Cooks-the-Meth. It seemingly has a very expensive pair of chemistry goggles glued to it.
```

description is too on-the-nose, remember to be physical with your description
a possible suggestion is:
```suggestion
  description: A bright orange lizard plushie dressed up as a chemist. It seemingly has a very expensive pair of chemistry goggles glued to it.
```

---

Updated.